packages/kit/src/core/build/index.js

```diff
@@ -252,9 +252,10 @@ async function build_server(
 			import { set_prerendering } from './runtime/env.js';
 			import * as user_hooks from ${s(app_relative(hooks_file))};
 
-			const template = ({ head, body }) => ${s(fs.readFileSync(config.kit.files.template, 'utf-8'))
+			const template = ({ head, body, lang }) => ${s(fs.readFileSync(config.kit.files.template, 'utf-8'))
 				.replace('%svelte.head%', '" + head + "')
-				.replace('%svelte.body%', '" + body + "')};
+				.replace('%svelte.body%', '" + body + "')
+				.replace('%svelte.lang%', '" + lang + "')};
```

Comment:
I don't know if we need to add special support for `%svelte.lang%` here. You can do this with hooks: https://github.com/sveltejs/kit/pull/641#issuecomment-806310491

Reply:
I've been using the hooks for replacing '%svelte.lang%' for my own svelte-lang solution but what I don't like about using hooks is that if you have a blog post or something that has the text '%svelte.lang%' anywhere, it too will get replaced with `lang` because we are replacing the response body, whereas this way we can ensure that we are only replacing text in the template.

Reply:
Nearly everything can be done without integrating i18n directly in SvelteKit, but a full integration could mean a better developer experience.